openshift/templates/pipeline/lup-api-build-deploy-dev.yaml

```diff
@@ -122,3 +122,16 @@ spec:
       taskRef:
         kind: ClusterTask
         name: openshift-client
+    - name: check-rollout-status
+      runAfter:
+        - trigger-deployment
+      taskSpec:
+        metadata: {}
+        spec: null
+        steps:
+          - computeResources: {}
+            image: 'image-registry.openshift-image-registry.svc:5000/openshift/cli:latest'
+            name: wait-for-rollout
+            script: |
```

Comment:
Is there a reason why this task doesn't use the `openshift-client` ClusterTask like the others?